templates/automotive/index.html

```diff
@@ -656,7 +656,6 @@ <h3 class="p-heading--2">Build the dealerships of the future</h3>
   <div class="u-fixed-width">
     <h2>Preferred by developers</h2>
     <p>Ubuntu is the <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">preferred OS for developers</a>, at 66% of working professionals &ndash; and growing.</p>
-    <p>Empower your team and attract the best talent.</p>
```

Comment:
This sentence is still on the doc, not sure if it was supposed to be removed or not

Reply:
Good catch, added back.